store/views.py

```diff
@@ -13,29 +14,28 @@ def index(request):
 def bookDetailView(request, bid):
     template_name = 'store/book_detail.html'
     context = {
-        'book': None, # set this to an instance of the required book
-        'num_available': None, # set this to the number of copies of the book available, or 0 if the book isn't available
+        'book': Book.objects.get(id=bid), # set this to an instance of the required book
```

Comment:
In case, book with this bid doesn't exist, your server returns an internal server error. Instead, it should return 404.